app/controllers/provider_interface/courses_controller.rb

```diff
@@ -0,0 +1,31 @@
+module ProviderInterface
+  class CoursesController < ProviderInterfaceController
+    before_action :set_application_choice
```

Comment:
could we add a before action to redirect to the application show page if the feature flag isn't on?